pom.xml

```diff
@@ -659,7 +659,7 @@
             <dependency>
                 <groupId>io.projectreactor</groupId>
                 <artifactId>reactor-bom</artifactId>
-                <version>Dysprosium-RELEASE</version>
+                <version>Dysprosium-SR6</version>
```

Comment:
why not SR7 while you are at it?

Reply:
because it has been released 8 hours ago

![Screenshot from 2020-04-27 23-17-26](https://user-images.githubusercontent.com/708318/80421805-5a28e180-88dd-11ea-8d42-065d20b3cbdb.png)

Reply:
I'm too bleeding edge :D